js/index.js

```diff
@@ -0,0 +1,28 @@
+// !! IMPORTANT README:
+
+// You may add additional external JS and CSS as needed to complete the project, however the current external resource MUST remain in place for the tests to work. BABEL must also be left in place. 
+
+/***********
+INSTRUCTIONS:
+  - Select the project you would 
```

Comment:
I Kindly suggest you remove useless comments as they do not make your code neat. Try as much as possible to summarize your comments into one-liners